config/credentials.yml.enc

```diff
@@ -1 +1 @@
-TqrAxzrYm2+C85eSjwC31qx5yEXwVMzP3aTk5OVE2qY6mc/qE9sJOJ88/X5sX42uupC3VOHZ4JKUIqAcio3hBvg1iXb2XQ6Y3y6NPAd6B5O0yEdb4+LRdBsBze2BEpn+KFowyvnXJQXmyx+ayAWv/zxae1mhEcRJDsyGNM8lH7NWVlpuRhio7zr+rKf3voW4yqqAJ4s7/3Uomhvs5NH3kbJ8azXZybOE9zsyQn/ja65hqnEzdawk+ET+eWXReVG5+jPeB/XDMkg2RgxLToQuLJQYijEmTOwSTSmTJeseW+/jOkxiOlTLWUUDzbmxkmEYH6FaoT7B6QNsGyBRlkCDq/GYAWSc30Fde/rdNgFeFMWeQo9rS5WONQ90WcdE/pgZ3VVGj9FuyoF5/N98DA0/xHtJZ/Z/K/BKxUaK--egWHj7Z23yZuOm9Q--gJ27n9X729OcYRvi0u868Q==
+TqrAxzrYm2+C85eSjwC31qx5yEXwVMzP3aTk5OVE2qY6mc/qE9sJOJ88/X5sX42uupC3VOHZ4JKUIqAcio3hBvg1iXb2XQ6Y3y6NPAd6B5O0yEdb4+LRdBsBze2BEpn+KFowyvnXJQXmyx+ayAWv/zxae1mhEcRJDsyGNM8lH7NWVlpuRhio7zr+rKf3voW4yqqAJ4s7/3Uomhvs5NH3kbJ8azXZybOE9zsyQn/ja65hqnEzdawk+ET+eWXReVG5+jPeB/XDMkg2RgxLToQuLJQYijEmTOwSTSmTJeseW+/jOkxiOlTLWUUDzbmxkmEYH6FaoT7B6QNsGyBRlkCDq/GYAWSc30Fde/rdNgFeFMWeQo9rS5WONQ90WcdE/pgZ3VVGj9FuyoF5/N98DA0/xHtJZ/Z/K/BKxUaK--egWHj7Z23yZuOm9Q--gJ27n9X729OcYRvi0u868Q==
```

Comment:
add new line please

Reply:
Unfortunately I can not because then master.key does not work